src/main/resources/templates/db_scripts/create-access-types-table.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE access_types
+DROP COLUMN IF EXISTS jsonb,
+ADD COLUMN IF NOT EXISTS credentials_id UUID NOT NULL CONSTRAINT access_types_kb_credentials_fkey REFERENCES kb_credentials,
+ADD COLUMN IF NOT EXISTS name VARCHAR (100) NOT NULL CONSTRAINT name_unique UNIQUE,
```

Comment:
we have a validation on name length - kb.ebsco.credentials.access.types.name.length.max=75

Reply:
But this setup can be changed in properties. I decide to add some buffer to have a possibility to increase length without changing a DB schema.